src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -620,22 +620,22 @@ export function getEventContent(event) {
       };
     case EVENT_TYPES.cavcDecision:
       return {
-        title: 'CAVC Decision',
+        title: 'CAVC decision',
         description: '',
       };
     case EVENT_TYPES.recordDesignation:
       return {
-        title: 'Designation of Record',
+        title: 'Designation of record',
```

Comment:
revert to
title: 'Designation of Record',

Reply:
Thanks @gnakm, updated.